how-to/samba/mount-cifs-shares-permanently.md

```diff
@@ -112,6 +112,12 @@ If you need to change the owner of a share, you'll need to add a **UID** (short
 //servername/sharename /media/windowsshare cifs uid=ubuntuusername,credentials=/home/ubuntuusername/.smbcredentials 0 0
 ```
 
+## Mount network folders with authd
+
+If you use Samba and authd at the same time, you must specify user and group mapping. Otherwise, you'd encounter permission issues due to mismatched user and group identifiers.
```

Comment:
```suggestion
If you use Samba and authd at the same time, you must specify user and group mapping. Otherwise, you will encounter permission issues due to mismatched user and group identifiers.
```